### PR TITLE
Refactor and add new text. Change icon, seperate icon styles

### PR DESCRIPTION
**Pull request by amanalili**

### CODE REVIEW

File: packages/edge-login-ui-rn/src/common/locales/strings/enUS.json [hunk 1]
```diff
@@ -134,7 +134,8 @@
   "create_account_error_message": "An error occurred creating your account. This may have been due to a slow network connection or network interruption. Please retry and if you receive another error that the account already exists, please restart the app and login with your username and password.",
   "try_again": "Try Again",
   "otp_reset_modal_header": "Secure Your Account",
-  "otp_reset_modal_message": "It is highly recommended that you enable Two Factor Authentication (2FA) to protect against unauthorized access to your account. Would you like to enable it now?",
+  "otp_reset_modal_message": "Two factor authentication (2FA) prevents unauthorized access from other devices, even if your username and password is compromised. You can scan a QR code or type in an authentication code to seamlessly authorize other devices. Would you like to enable 2fa?",
```
Comment (paullinator) [inline packages/edge-login-ui-rn/src/common/locales/strings/enUS.json]: Capitalize "2FA"

File: packages/edge-login-ui-rn/src/common/locales/strings/enUS.json [hunk 2]
```diff
@@ -134,7 +134,8 @@
...
   "otp_reset_modal_dont_ask": "Don't ask again",
-  "otp_authentication_header": "Authentication Code"
+  "otp_authentication_header": "2FA has been enabled",
+  "otp_authentication_message": "2FA has been enabled. A unique authentication code will be generated. If you lose your phone or uninstall the app, it will take 7 days to disable 2FA and access your account from another device with the following authentication code.\n\n%s"
```
Comment (paullinator) [inline packages/edge-login-ui-rn/src/common/locales/strings/enUS.json]: Change to "A unique authentication code has been generated."

and

"from another device without the following authentication code"

File: packages/edge-login-ui-rn/src/common/locales/strings/enUS.json (hunk 1 above)
Comment (amanalili) [reply packages/edge-login-ui-rn/src/common/locales/strings/enUS.json]: updated

File: packages/edge-login-ui-rn/src/common/locales/strings/enUS.json (hunk 2 above)
Comment (amanalili) [reply packages/edge-login-ui-rn/src/common/locales/strings/enUS.json]: updated

File: packages/edge-login-ui-rn/src/common/locales/strings/enUS.json (hunk 1 above)
Comment (swansontec) [reply packages/edge-login-ui-rn/src/common/locales/strings/enUS.json]: Second "2fa" instance is still lower case.

**Pull request merged**